### PR TITLE
[playground] Adding workflow to Update K8s Cleanup Script

### LINKED ISSUE #7818: [playground] Add workflow to Update K8s Cleanup Script
#### Current Behavior

Recently a new script was added to ensure that k8s cluster is cleanup nightly.
Though the script inside VM is not set to automatically update when a change is made to https://github.com/meshery/meshery/blob/master/.github/scripts/playground-cleanup.sh 

#### Desired Behavior

When a change is made in https://github.com/meshery/meshery/blob/master/.github/scripts/playground-cleanup.sh then ensure that a new workflow is used to update the `playground-cleanup.sh` script places in c3-medium-x86-01-meshery VM at the root directory.

#### Implementation

1. Add a new workflow that watches for `/.github/scripts/playground-cleanup.sh`
2. SSH into c3-medium-x86-01-meshery
3. Update the script in `/playground-cleanup.sh` of root dir of c3-medium-x86-01-meshery with the new change in the master branch `/.github/scripts/playground-cleanup.sh`

#### Acceptance Tests

Reference:
1. Original post: https://github.com/meshery/meshery/pull/7799#issue-1743439241
2. https://github.com/meshery/meshery/blob/master/install/playground/readme.md

---
#### Contributor [Guides](https://docs.meshery.io/project/contributing) and [Handbook](https://layer5.io/community/handbook)
- 🛠 [Meshery Build & Release Strategy](https://docs.meshery.io/project/contributing/build-and-release)
- 📚 [Instructions for contributing to documentation](https://github.com/meshery/meshery/blob/master/CONTRIBUTING.md#documentation-contribution-flow)
   - Meshery documentation [site](https://docs.meshery.io/) and [source](https://github.com/meshery/meshery/tree/master/docs)
- 🎨 Wireframes and designs for Meshery UI in [Figma](https://www.figma.com/file/SMP3zxOjZztdOLtgN4dS2W/Meshery-UI)
- 🙋🏾🙋🏼 Questions: [Discussion Forum](https://discuss.layer5.io) and [Community Slack](http://slack.layer5.io)

### PR DESCRIPTION
**Pull request by suhail34**
**Notes for Reviewers**

This PR fixes #7818

**[Signed commits](https://github.com/meshery/meshery/blob/master/CONTRIBUTING.md#signing-off-on-commits-developer-certificate-of-origin)**
- [ ] Yes, I signed my commits.

### CODE REVIEW

**Comment by leecalcote** [comment]
@Freedisch

**Comment by leecalcote** [comment]
@Freedisch

File: .github/workflows/update-script-in-vm.yaml [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+    
+        steps:
+        - name: Update Script in VM
+          uses: appleboy/ssh-action@master
+          with:
+            host: 147.75.47.9
+            username: root
+            key: ${{ secrets.METAL_SSH_KEY }}
+            script: |
+                cd
```
Comment (Revolyssup) [inline .github/workflows/update-script-in-vm.yaml:20]: Can you remove this `cd` ?

**Review by Revolyssup — CHANGES_REQUESTED** [review]

**Comment by Revolyssup** [comment]
Just the above requested change and everything LGTM

File: .github/workflows/update-script-in-vm.yaml (hunk 1 above)
Comment (suhail34) [reply .github/workflows/update-script-in-vm.yaml:20]: I was trying to be conscious so that it gets into root dir will remove it

**Review by Revolyssup — APPROVED** [review]

**Pull request merged**